src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -16,7 +16,7 @@ Some highlights include:
 - **[UI-agnostic](/en/guides/framework-components/):** Supports React, Preact, Svelte, Vue, Solid, Lit, HTMX, web components, and more.
 - **[Server-first](/en/basics/rendering-modes/):** Moves expensive rendering off of your visitors' devices.
 - **[Zero JS, by default](/en/basics/astro-components/):** Less client-side JavaScript to slow your site down.
-- **[Content collections](/en/guides/content-collections/):** Organize, validate, and provide TypeScript type-safety for your Markdown content.
+- **[Content collections](/en/guides/content-collections/):** Organize, validate, and provide TypeScript type-safety for your JSON or Markdown content.
```

Comment:
should `YAML` be appended to this list as well? I see the format is called data-serialization

Reply:
I don't see an example of YAML on this docs page: https://docs.astro.build/en/guides/content-collections/

Unless you're considering Markdown frontmatter YAML?

If you can indeed have `.yaml` files as an alternative to `.json` files, perhaps there should be a table that identifies which extensions will work for which types of collections.